airbyte-integrations/connectors/source-braintree/acceptance-test-config.yml

```diff
@@ -0,0 +1,29 @@
+connector_image: airbyte/source-braintree:dev
+tests:
+  spec:
+    - spec_path: "integration_tests/spec.json"
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
+  discovery:
+    - config_path: "secrets/config.json"
+  basic_read:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog.json"
+      empty_streams: ["subscription_stream"]
+      expect_records:
+        path: "integration_tests/expected_records.txt"
+        extra_fields: no
+        exact_order: no
+        extra_records: yes
+  incremental:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog_incremental.json"
+      future_state_path: "integration_tests/abnormal_state.json"
+      cursor_paths:
+        charges: ["created_at", "received_date"]
```

Comment:
Why do we need `cursor_paths` here? In `configured_catalog_incremental.json` you have `default_cursor_field` which should be enough, right? Also, I checked `sample_state.json` file and there you have `<stream_name>` -> `<cursor_field>` -> `<value>` and not, for example, `<stream_name>` -> `<some_field>` -> `<cursor_field>` -> `<value>` .

Reply:
removed this field.